pkg/network/vmispec/interface.go

```diff
@@ -174,10 +174,17 @@ func FilterInterfacesByNetworks(interfaces []v1.Interface, networks []v1.Network
 
 func BindingPluginNetworkWithDeviceInfoExist(ifaces []v1.Interface, bindingPlugins map[string]v1.InterfaceBindingPlugin) bool {
 	for _, iface := range ifaces {
-		if iface.Binding != nil {
-			if binding, exist := bindingPlugins[iface.Binding.Name]; exist && binding.DownwardAPI == v1.DeviceInfo {
-				return true
-			}
+		if BindingPluginDeviceInfoIface(iface, bindingPlugins) {
+			return true
+		}
+	}
+	return false
+}
+
+func BindingPluginDeviceInfoIface(iface v1.Interface, bindingPlugins map[string]v1.InterfaceBindingPlugin) bool {
```

Comment:
I do not understand from the name that it returns a bool.
Perhaps `HasBindingPluginDeviceInfo`.